### DISCUSSION TITLE
How to overwrite the "app.[contenthash].js" 's filename?

### DISCUSSION THREAD

**Discussion by zhimng** [nuxt/nuxt#9764] [Questions (Nuxt 2)] [answered]
```javascript
optimization: {
  splitChunks: {
    cacheGroups: {
      vue: {
        enforce: true,
        chunks: 'initial',
        test: /vue/,
        priority: 100,
        filename: 'vue.js',
      },
      moment: {
        enforce: true,
        test: /moment/,
        priority: 90,
        filename: 'moment.js',
      },
      default: {
        chunks: 'initial',
        filename: 'app.js',
        enforce: true,
      },
    },
  },
},
```

I want to change the filename in chunks, but it still get `app.[contenthash].js`,  i use the `splitChunks` to split vendors from `app.[contenthash].js` and the filename is `app.js`, but the `app.js` is not overwrite the `app.[contenthash].js`, Am I configured wrong? 

![image](https://user-images.githubusercontent.com/20208875/132133521-8ee772a8-6b0f-4933-a39d-1c0944e4c0f9.png)

**Comment by TheAlexLichter** [comment, marked as answer]
You can use the [build.filenames](https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-build#filenames) functionality ☺️

**Reply by zhimng** [reply]
For some specific reason, I just want to remove the `contenthash` from the `javascript` file imported in `index.html` instead of removing the `contenthash` from all the chunks.
